python/seldon_core/metrics.py

```diff
@@ -184,6 +184,16 @@ def generate_metrics(self):
             exposition.CONTENT_TYPE_LATEST,
         )
 
+    def clear(self):
+        """
+        Clear all metrics from current worker.
+        """
+        worker_id = self.worker_id_func()
+        logger.debug("Clearing metrics from worker #{worker_id}")
```

Comment:
hmm... is the `#{...}` some new thing? shouldn't it be `f-string`?

Reply:
Good catch. Let me update that.